src/symbols.rs

```diff
@@ -470,6 +470,56 @@ pub mod border {
         horizontal_top: QUADRANT_TOP_HALF,
         horizontal_bottom: QUADRANT_BOTTOM_HALF,
     };
+
+    /// Solid border set
+    ///
+    /// The border is created by using full blocks for all sides.
+    ///
+    /// ```text
+    /// ████
+    /// █xx█
+    /// █xx█
+    /// ████
+    pub const FULL: Set = Set {
+        top_left: block::FULL,
+        top_right: block::FULL,
+        bottom_left: block::FULL,
+        bottom_right: block::FULL,
+        vertical_left: block::FULL,
+        vertical_right: block::FULL,
+        horizontal_top: block::FULL,
+        horizontal_bottom: block::FULL,
+    };
+
+    /// Empty border set
+    ///
+    /// The border is created by using empty strings for all sides.
+    ///
+    /// This is useful for ensuring that the border style is applied to a border on a block with a
+    /// title without actually drawing a border.
```

Comment:
Maybe we should enforce an empty border when a title is defined, but no border is explicitly defined? Sounds like a bug that should be fixed internally rather than adding a workaround for external use?

Reply:
It's sort of an edge case (pun intended). I think it probably does need to be treated as a bug, but I'm not 100% sure of that. In the meantime, these are useful in addressing the problem more simply.